packages/@lwc/engine-core/src/framework/profiler.ts

```diff
@@ -118,4 +191,35 @@ const profilerControl = {
     detachDispatcher,
 };
 
-export { logOperationStart, logOperationEnd, trackProfilerState, profilerControl };
+function opIdForGlobalMeasurementPhase(phase: GlobalMeasurementPhase) {
+    return phase === GlobalMeasurementPhase.HYDRATE
+        ? OperationId.globalHydrate
+        : OperationId.globalRehydrate;
+}
+
+function logGlobalOperationStart(phase: GlobalMeasurementPhase, vm?: VM) {
```

Comment:
There is something strange here IMO, `logOperationStart` accepts a `OperationId`, `logGlobalOperationStart` accepts a `GlobalMeasurementPhase`. Especially when `OperationId` also include global measure.

Ideally, both methods should accept `OperationId`.

Reply:
We could combine the `GlobalMeasurementPhase` and `OperationId` enums into a single enum. But yeah, I'm happy to punt this to another PR.